Compilation error MSVC 16.4.1

Microsoft Visual Studio Community 2019
Version 16.4.1
C++20 enabled
```
cpptaskflow\taskflow\core/executor.hpp(864,41): error C3538: in a declarator-list 'std::scoped_lock' must always deduce to the same type
cpptaskflow\taskflow\core/executor.hpp(864,41): message : could be 'std::scoped_lock<>'
cpptaskflow\taskflow\core/executor.hpp(864,41): message : or       'std::scoped_lock<std::mutex>'
```

P.S. Works fine with the previous version of MSVC.

--- a/taskflow/core/executor.hpp
+++ b/taskflow/core/executor.hpp
@@ -861,13 +861,13 @@ std::future<void> Executor::run_until(Taskflow& f, P&& pred, C&& c) {
 
 // Procedure: _increment_topology
 inline void Executor::_increment_topology() {
-  std::scoped_lock lock(_topology_mutex);
+  std::scoped_lock<std::mutex> lock(_topology_mutex);
   ++_num_topologies;
 }
 
 // Procedure: _decrement_topology_and_notify
 inline void Executor::_decrement_topology_and_notify() {
-  std::scoped_lock lock(_topology_mutex);
+  std::scoped_lock<std::mutex> lock(_topology_mutex);
   if(--_num_topologies == 0) {
     _topology_cv.notify_all();
   }

